tools/packaging/static-build/ovmf/build-ovmf.sh

```diff
@@ -0,0 +1,85 @@
+#!/bin/bash
+#
+# Copyright (c) 2022 IBM
+#
+# SPDX-License-Identifier: Apache-2.0
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+script_dir="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+
+source "${script_dir}/../../scripts/lib.sh"
+
+ovmf_repo="${ovmf_repo:-}"
+ovmf_dir="edk2"
+ovmf_version="${ovmf_version:-}"
+kata_version="${kata_version:-}"
+
+set +u
+ovmf_build="${ovmf_build:-sev}"
+architecture="X64"
+toolchain="GCC5"
+build_target="RELEASE"
+
+if [ -z "$ovmf_repo" ]; then
+       info "Get ovmf information from runtime versions.yaml"
+       ovmf_url=$(get_from_kata_deps "externals.ovmf.url" "${kata_version}")
+       [ -n "$ovmf_url" ] || die "failed to get ovmf url"
+       ovmf_repo="${ovmf_url}.git"
+fi
+
+[ -n "$ovmf_repo" ] || die "failed to get ovmf repo"
+
+if [ "${ovmf_build}" == "sev" ]; then
+       [ -n "$ovmf_version" ] || ovmf_version=$(get_from_kata_deps "externals.ovmf.sev.version" "${kata_version}")
+fi
+[ -n "$ovmf_version" ] || die "failed to get ovmf version or commit"
+
+info "Build ${ovmf_repo} version: ${ovmf_version}"
+
+[ -d "${ovmf_dir}" ] || git clone "${ovmf_repo}"
```

Comment:
Even if `${ovmf_dir}` exists it is not guarantee to be the same sources as from `${ovmf_repo}`. My suggestion is to delete and then clone. Actually, I think it would be better to use a temporary directory for the build (and properly clean up it afterwards), for example:

```
....
build_root=$(mktemp -d)
cleanup() {
    if [ "${DEBUG:-}" == true ]; then
       info "Not deleted build root directory: ${build_root}"
       return
    fi
    rm -rf "${build_root}"
}
....
trap cleanup EXIT
```